### PR TITLE
Incremented minor version number, added to news

### PR DESCRIPTION
**Pull request by AayushSabharwal**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "Agents"
 uuid = "46ada45e-f475-11e8-01d0-f70cc89e6671"
 authors = ["Tim DuBois", "George Datseris", "Ali Vahdati"]
-version = "4.2.5"
+version = "4.2.6"
```
Comment (Datseris) [inline Project.toml:4]: Please see https://semver.org/ . You have to increment the MINOR version

File: docs/src/index.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ To get started, please read the [Tutorial](@ref) page.
       * New function `ensemblerun!` which replaces using `replicates` in `run!`
       * New documentation page "Performance Tips"
       * Parallelization of `paramscan`
+      * New functions `dump_to_csv` and `populate_from_csv!` to save and load agent data to and from CSV files
```
Comment (Datseris) [inline docs/src/index.md:18]: The old news have to be removed though ;) The point of this block is to bring the users attention to something really new. It is not to keep a changelog. We have a changelog that lists all incremental changes :)

File: Project.toml (hunk 1 above)
Comment (AayushSabharwal) [reply Project.toml:4]: Didn't know that, thanks. I'll keep it in mind

Comment (AayushSabharwal) [reply Project.toml:4]: This would become 4.3.0, right?

**Comment by AayushSabharwal** [comment]
I updated the printed message in `src/Agents.jl` too

**Comment by Datseris** [comment]
I wouldn't do any message printing yet. We should have a message printing once the "full" AgentsIO functionality is present, which includes the checkpoints. BTW we need to resolve #493 before we can do another update message.

**Comment by AayushSabharwal** [comment]
I'll revert the message printing then, and just bump the `version_number` in that file.

Once this is merged, should I create a new release or wait for #493 to be resolved?

**Comment by Datseris** [comment]
Nah we just tag a new release, #493 only needs to be resolved when we want an update message.

**Review by Datseris — APPROVED** [review]

File: src/Agents.jl [hunk 3]
```diff
@@ -58,7 +58,7 @@ export Models
 
 # Update message:
 display_update = true
-version_number = "4.2"
+version_number = "4.3"
```
Comment (Datseris) [inline src/Agents.jl:61]: If this is changed, the update message will trigger. If we don't want to trigger an update message, then either set `display_update = false` or leave the version unchanged.

**Review by Libbum — APPROVED** [review]

**Comment by Datseris** [comment]
do not forget to delete the branches after merge ⬆️

**Pull request merged**